classes/form/edit_entry.php

```diff
@@ -35,4 +50,6 @@ public function definition() {
 
         $this->add_action_buttons();
     }
+
```

Comment:
It's a good habit to not let unrelated changes like whitespace slip into a diff like this